modules/stanford_profile_styles/lib/scss/component/node/stanford_page.scss

```diff
@@ -320,6 +320,11 @@
       }
     }
   }
+
+  .su-event-list-item__date {
```

Comment:
Why is this necessary if the same styles are applied in [stanford_events](https://github.com/SU-SWS/stanford_events/pull/76/files#diff-0cde1a1f3737bd25dc63d5c13d361da7c2a74abf976f2ee9bb957fd19f4a279bR4)

I followed the steps and with only the stanford_events PR I believe it works for me.

Reply:
it is necessary for the default list version in a paragraph. If you add a list in the default style, you would see this.
![image](https://user-images.githubusercontent.com/1187335/123838400-d263ac00-d8c0-11eb-9129-aa1289bf41d5.png)

I thought about adding the `.ptype-stanford-lists` to the events file, but we've talked before about keeping them separate.

Reply:
What about just making it less specific. if i remove `.su-card` from the [scss](https://github.com/SU-SWS/stanford_events/pull/76/files#diff-0cde1a1f3737bd25dc63d5c13d361da7c2a74abf976f2ee9bb957fd19f4a279bR3) it applies the same styles to both displays.

Reply:
True but then it does this on the /events list pages. 
![image](https://user-images.githubusercontent.com/1187335/123841801-db567c80-d8c4-11eb-9bb4-f4c1439111d9.png)
Better to override it just on the list page? I could look at it that way too.

Reply:
Oh gotcha. This is really why we need the `@container` css if it were widely available. Darn you CSS!